src/Sarif.Viewer.VisualStudio.UnitTests/TestData/BinaryTestFiles/NonBinaries/csharpfile.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace AnalysisTestProject1
+{
+    internal class Class1
+    {
+    }
+}
```

Comment:
What is the function of this empty class?

Reply:
this is a test .cs file (non-binary file) along with the other binaries test files, they are used for unit tests for binary scenarios. The content should not matter, I just put test C# code to simulate a C# code file.